### ISSUE TITLE
Arrow Fix

### ISSUE THREAD

**Issue by HadesHerald** [codecombat/codecombat#2193]
The arrow (as commented below) is a little buggy and extremely seizure inducing when you have any manner of lag.  A possible is to simply highlight the border of the image.
It would be a better version of 
![image](https://cloud.githubusercontent.com/assets/10787055/5990229/89396778-a964-11e4-90ff-e4c272fdb268.png)

instead of 

![image](https://cloud.githubusercontent.com/assets/10787055/5990231/b4340442-a964-11e4-8104-ee61da04799a.png)

This minimizes induced epilepsy and has potential to look much nicer.

**Comment by HadesHerald** [comment]
![example item](https://cloud.githubusercontent.com/assets/10787055/5990313/d3316904-a967-11e4-8b27-83c1d0dfe433.png)
Like this kind of pretty

**Comment by nwinter** [comment]
Hmm; it looks nicer, but many players will not see it. We have made the arrow bigger and bigger many times until in our playtests it was visible enough. What could we do that had around as much visual weight but that wasn't an epilepsy trigger?

**Comment by popey456963** [comment]
What about making it static, and fade, rather than bounce in?  That way we'd keep most of it's visual weight, but it wouldn't give anyone epilepsy?

**Comment by nwinter** [comment]
Both of those changes would reduce its prominence a lot, so you'd need something else to balance it out. At one point we tried having it make a bouncing noise, but that was so annoying we had to get rid of it.

**Comment by HadesHerald** [comment]
Ever considered a trumpet noise and static arrow?

**Comment by nwinter** [comment]
Yeah. We can't rely too much on the noise, because many players are playing without sound.

**Comment by nwinter** [comment]
Hmm; maybe also using the new particle effects system can help to reduce the need for the arrow to be so annoying?
